Integrations/integration-C2sec.yml

```diff
@@ -0,0 +1,236 @@
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {}
+                }
+       else:
+           return ('Error querying for issues [%d] - %s' % (call.status_code, call.text))
+
+
+    def rescan(domain):
+       params = {'apikey': demisto.params()['apikey'], 'domain': domain}
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(demisto.params()['endpointURL'] + 'iRiskRescanCompany', headers=headers, params=params)
+       if call.status_code == requests.codes.ok:
+         md = tableToMarkdown('Rescan Results', call.json())
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {'c2secScanRequest': call.json()}
+                }
+       else:
+           return ('Error rescanning the company  [%d] - %s' % (call.status_code, call.text))
+
+
+    def queryComponent(domain, component):
+       params = {'apikey': demisto.params()['apikey'], 'domain': domain, 'component': component }
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(demisto.params()['endpointURL'] + 'iRiskQueryComponentData', headers=headers, params=params)
+       if call.status_code == requests.codes.ok:
+         md = tableToMarkdown('Query Component Result', call.json()['data'])
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {}
+                }
+       else:
+           return ('Error querying for component [%d] - %s' % (call.status_code, call.text))
+
+    def createIncident(c2sec, domain, component):
+       occured = datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S.000000Z")
+       keys = c2sec.keys()
+       labels = []
+       name = 'test'
+       if component == 'credential':
+         name = 'C2SEC Leaked Credentials'
+       elif component == 'application':
+           name = 'C2SEC Application Vulnerabilities'
+       elif component == 'network':
+           name = 'C2SEC Network Services Exposed'
+       for i in range(len(keys)):
+          if isinstance(c2sec[keys[i]], list):
+            labels.append({'type': keys[i], 'value': json.dumps(c2sec)})
+          else:
+              labels.append({'type': keys[i], 'value': str(c2sec[keys[i]])})
+       return {
+            'name': name,
+            'labels': labels,
+            'rawJSON': json.dumps(c2sec),
+            'occurred': occured
+        }
+
+
+    def return_error(data):
```

Comment:
Not needed, we have it under `CommonServerPython` so you can just remove this function from here and it will still work

Reply:
fixed